app/components/trainee_name/view.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module TraineeName
+  class View < GovukComponent::Base
+    attr_reader :trainee
+
+    def initialize(trainee)
+      @trainee = trainee
+    end
+
+    def display_text
+      full_name || t("views.trainees.show.draft")
+    end
+
+  private
+
+    def full_name
+      [
+        trainee.first_names,
+        trainee.middle_names,
+        trainee.last_name,
+      ].select(&:present?).join(" ").presence
+    end
```

Comment:
Should this go in to the model?

Reply:
Short name for this please, not full name.